Content.Server/Explosion/EntitySystems/ClusterGrenadeSystem.cs

```diff
@@ -122,8 +123,9 @@ public override void Update(float frameTime)
 
                 var clusterEv = new CMClusterSpawnedEvent(_spawned);
                 RaiseLocalEvent(uid, ref clusterEv);
-                // delete the empty shell of the clusterbomb
-                Del(uid);
+                //del or qdel causes lag out the ass for shrapnel type clusters.
+                var despawn = AddComp<TimedDespawnComponent>(uid);
+                despawn.Lifetime = 0.01f;
```

Comment:
Can you explain this change? We're not sure how this helps performance

(Also if it does help maybe it's a good upstream change as well?)

Reply:
discussed on discord, it was an issue with the component storing the deleted uid in one of the automatically networked fields